Jenkinsfile

```diff
@@ -168,6 +167,27 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
       }
     }
   )
+  pipeline.createStage(
```

Comment:
Tom had switched the build pipeline to Github Actions. This file is a backup reference for now. We may need to modify somewhere here https://github.com/zowe/zowe-install-packaging/blob/staging/.github/workflows/build-packaging.yml#L289-L324 instead.